### PR TITLE
Library Update 2023

### PR DESCRIPTION
**Pull request by Simon-Heck**

### CODE REVIEW

**Comment by anthonyjclark** [comment]
Let me know when this is ready for review.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# Created by https://www.toptal.com/developers/gitignore/api/python,visualstudiocode
```
Comment (anthonyjclark) [inline .gitignore:1]: Why do you keep changing all of the `.gitignore` files?

File: ue5env/__init__.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+    def isconnected(self):
+        """Is the program connected to Unreal"""
+        return self.ue5.isconnected()
+
+    def reset(self):
+        """Reset robot to start location. Interacts with UE5 Blueprint."""
+        self.ue5.request(f"vset /action/keyboard backspace 0.1")
+
+    def getCameraLocation(self, cameraID: int = 0) -> tuple[float, float, float]:
```
Comment (anthonyjclark) [inline ue5env/__init__.py]: Use snake case.

**Review by anthonyjclark — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (Simon-Heck) [reply .gitignore:1]: I believe it was automated, I haven't touched the gitignore in a while

**Pull request merged**